Kernel/System/Package.pm

```diff
@@ -3908,7 +3908,7 @@ sub _CheckModuleRequired {
                 $Kernel::OM->Get('Kernel::System::Log')->Log(
                     Priority => 'error',
                     Message  => "Sorry, can't install package, because module "
-                        . "$Module->{Content} v$Module->{Version} is required "
+                        . "$Module->{Content} " . ( $Module->{Version} ? "v$Module->{Version}" : "" ) . " is required "
```

Comment:
Being a bit picky here. Could you eliminate the extra space when there is no version?

Reply:
Ah, yes, you're right - I thought that I had positioned the quotes to do so, but I overlooked it ;)